task/minio-cli/0.1/tests/secret.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: key-tokens
+stringData:
+  access-token: Q3AM3UQ867SPQQA43P2F
+  secret-token: zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
```

Comment:
Just a question how long these secretes are gonna be valid ???

Reply:
@msmagnanijr  instead of using these credentials can we just create our own `minio` deployment for testing ?
probably you can follow the approach as followed in [jenkins](https://github.com/tektoncd/catalog/tree/main/task/jenkins/0.1/tests) task.

Reply:
sure! I’ll try to do that and then I’ll come back to you.